modules/ding_seo/ding_seo.module

```diff
@@ -4,3 +4,6 @@
  * @file
  * Ding SEO module file.
  */
+
+// includes custom ding_seo tokens
+include_once 'ding_seo.tokens.inc';
```

Comment:
This file should be included automatically when token related hooks are invoked.

Reply:
Will remove this as suggested.